src/components/screens/votingQueue/form/voteRow.js

```diff
@@ -61,13 +61,17 @@ const VoteRow = ({
         </div>
       </div>
       <span className={`${styles.oldAmountColumn} ${styles.centerContent}`}>
-        <LiskAmount val={confirmed} token={token} />
+        {confirmed ? (
+          <LiskAmount val={confirmed} token={token} />
```

Comment:
I wonder if it would be helpful to add the hyphen fallback to the `LiskAmount` itself. So in case the value was undefined or any invalid type, it could show a hyphen and log a warning message.